docs/installation.rst

```diff
@@ -328,7 +328,7 @@ In Fedora, the command is::
 
 .. Note:: ``redhat-rpm-config`` is required on Fedora 23, but not earlier versions.
 
-Prerequisites are installed on **Ubuntu 16.04 LTS** with::
+Prerequisites for **Ubuntu 16.04 LTS or later** are installed with::
```

Comment:
```suggestion
Prerequisites for **Ubuntu 16.04 LTS - 20.04 LTS** are installed with::
```
Just a thought, since at some point in the future, these packages will stop being available for the latest Ubuntu.

Reply:
works for me.